scenario-compiler/src/main/scala/pl/touk/nussknacker/engine/spel/internal/DefaultSpelConversionsProvider.scala

```diff
@@ -31,6 +34,13 @@ class DefaultSpelConversionsProvider extends SpelConversionsProvider {
     service.addConverter(new ObjectToArrayConverter(service))
     // For purpose of concise usage of numbers in spel templates
     service.addConverter(classOf[Number], classOf[String], (source: Number) => source.toString)
+    service.addConverter(
+      classOf[GenericRecord],
+      classOf[java.util.Map[_, _]],
+      (r: GenericRecord) => {
```

Comment:
Unfortunately, we must do it with reflection as well - see the comment in build.sbt.